packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -186,6 +186,21 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the component should render in compact mode. Set to false by default */
   compact?: boolean;
+
+  /**
+  * Boolean value to enable render page caching. This is an experimental performance optimization 
+  * that is off by default.
+  * @defaultValue false
+  */
+  usePageCache?: boolean;
+  
+  /**
+   * Optional callback to determine whether the list should be rendered in full, or virtualized.
+   * Virtualization will add and remove pages of items as the user scrolls them into the visible range.
+   * This benefits larger list scenarios by reducing the DOM on the screen, but can negatively affect performance for smaller lists.
+   * The default implementation will virtualize when this callback is not provided.
+   */
+  onShouldVirtualize?: (props: IListProps) => boolean;
```

Comment:
@dzearing @jijizz we already have `listProps` that can be used to pass properties to the underlying list.. can we keep these pass-through properties out of the DetailsList props?

Reply:
I'm all for perf experiments, but it'll be hard to remove these properties in the future.